cypress/fixtures/rosa/RosaClusterClassicCreatePublic.json

```diff
@@ -67,13 +66,45 @@
             "InvalidOrganizationIDError": "Organization ID must be an alpha numeric with no symbols"
           }
         }
+      },
+      "RolesAndAccess": {
+        "dedicated-admin-user": {
+          "UserID": "ocmcypress1",
+          "Group": "dedicated-admins"
+        },
+        "cluster-admin-user": {
+          "UserID": "ocmcypress2",
```

Comment:
Also, please create an entry for a test data field that ends with ****@redhat.com